Cubical/Data/Prod/Properties.agda

```diff
@@ -48,12 +48,18 @@ private
   testrefl = refl
 
 -- equivalence between the sigma-based definition and the inductive one
-A×B≡A×ΣB : A × B ≡ A ×Σ B
-A×B≡A×ΣB = isoToPath (iso (λ { (a , b) → (a , b)})
+A×B≃A×ΣB : A × B ≃ A ×Σ B
+A×B≃A×ΣB = isoToEquiv (iso (λ { (a , b) → (a , b)})
                           (λ { (a , b) → (a , b)})
                           (λ _ → refl)
                           (λ { (a , b) → refl }))
 
+A×B≡A×ΣB : A × B ≡ A ×Σ B
```

Comment:
Btw, what is the point of having both of these definitions of products around? Having products defined using Sigma seems like the better option as we can easier inherit a lot of theory.

Reply:
I tend to agree, this is what `Data.Prod.Base` says:
```agda
-- If × is defined using Σ then transp/hcomp will be compute
-- "negatively", that is, they won't reduce unless we project out the
-- first of second component. This is not always what we want so the
-- default implementation is done using a datatype which computes
-- positively.
```
I have no experience with this so I can't speak to how important it is.

Reply:
Oh, right... Sometimes I write some good comments! I just have to remember to read them...